index.js

```diff
@@ -41,4 +50,16 @@ module.exports = (fn, limit, interval) => {
 			}
 		});
 	};
+
+	throttled.abort = () => {
+		for (const id of timeouts) {
+			clearTimeout(id);
+		}
+		for (const x of queue) {
+			x.reject(new Error('Throttled function aborted'));
```

Comment:
Can you make this a custom AbortError so users are able to ignore it easily if wanted?

```js
class AbortError extends Error {
	constructor(message) {
		super(message);
	}
}
```

And expose `AbortError` on the `module.export` and in the docs.

What do you think?